### PR TITLE
added ability to set parameters of a test

### PR DESCRIPTION
**Pull request by osherdp**
Added a new argument 'parameters' to the 'start_test_item' methods.
Bumped the version to 3.1.0, as it's a new feature that doesn't break any compatibility.

### CODE REVIEW

**Comment by osherdp** [comment]
@krasoffski / @frizzby, can you review the changes?

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 setup(
     name='reportportal-client',
     packages=find_packages(),
-    version='3.0.0',
+    version='3.1.0',
```
Comment (ailjushkin) [inline setup.py]: Are you sure about this changes does not break backward compability and work similarly for 3.x and 4.x Report Portal API?

**Comment by ailjushkin** [comment]
I have no any more remarks about this.

File: setup.py (hunk 1 above)
Comment (osherdp) [reply setup.py]: I'll do the tests and return with some answers

Comment (osherdp) [reply setup.py]: I tested the code on both RP 3 and RP 4.
The 'parameters' argument is simply being ignored in RP 3

Comment (osherdp) [reply setup.py]: But do you want me to update the version to 3.2.0?
There's already 3.1.0 on PyPI

Comment (krasoffski) [reply setup.py]: The correct way might be `3.0.1` and than `3.1.0`.

To be honest, this is bugfix, thus according to `semver` it should be 3.2.0 (as you mentioned).
Also, please, do not forget to update `download_url='https://github.com/reportportal/client-Python/tarball/3.0.0',` to right download url.

Comment (osherdp) [reply setup.py]: Nice catch. Fixed

**Comment by ailjushkin** [comment]
Thanks, merged

**Comment by krasoffski** [comment]
@DzmitryHumianiuk could you please merge this PR?

**Comment by osherdp** [comment]
:thinking:

**Comment by ailjushkin** [comment]
@krasoffski can you please try to close your review, seems like you did not finish it and thus the merge button is blocked. After finishing your review, it could stay possible to merge it by yourself.

**Comment by osherdp** [comment]
Hey,
Any news?

**Comment by DzmitryHumianiuk** [comment]
done

**Comment by osherdp** [comment]
Thanks

**Pull request merged**